src/clock-weather-card.ts

```diff
@@ -223,6 +226,7 @@ export class ClockWeatherCard extends LitElement {
         <clock-weather-card-today-right-wrap>
           <clock-weather-card-today-right-wrap-top>
             ${this.config.hide_clock ? weatherString : localizedTemp ? `${weatherString}, ${localizedTemp}` : weatherString}
+            ${!this.config.hide_humidity && localizedHumidity ? html`<div>${localizedHumidity} Humidity</div>` : ''}
```

Comment:
do we actually require the `div` at this point?

Reply:
I'm horrible with HTML, and this is my first time using TS also... so... probably not :D  How about a <br>? was just looking for a way to get it to the next line so it wouldn't clip on smaller dashboards.